ddev/src/ddev/cli/test/__init__.py

```diff
@@ -247,7 +247,7 @@ def test(
                 # In order to handle multiple environments the report files must contain the environment name.
                 # Hatch injects the `HATCH_ENV_ACTIVE` environment variable, see:
                 # https://hatch.pypa.io/latest/plugins/environment/reference/#hatch.env.plugin.interface.EnvironmentInterface.get_env_vars
-                command.extend(('--junit-xml', f'.junit/test-{"e2e" if e2e else "unit"}-$HATCH_ENV_ACTIVE.xml'))
+                command.extend(('--junit-xml', f'junit/test-{"e2e" if e2e else "unit"}-$HATCH_ENV_ACTIVE.xml'))
```

Comment:
IF we are modifying this we need to also modify both the workflow files that read this coverage file and gitignore to be processed later and gitignore to ensure we do not commit it by mistake:

- [test-target](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/test-target.yml#L401)
- [test-fips](https://github.com/DataDog/integrations-core/blob/master/.github/workflows/test-fips-e2e.yml)
- [gitignore](https://github.com/DataDog/integrations-core/blob/master/.gitignore)

Reply:
Yea, I noticed that but thanks for double-checking! I believe the preferred way for this fix is to explicitly pass the `DD_SERVICE` so I've been looking into that.

Reply:
I couldn't find a good way to inject the `DD_SERVICE` so going with this 🙇